### PR TITLE
[vNext] updated to use dnx.rc1-update-1 with a hack for the  A supported MIME…

### PR DESCRIPTION
**Pull request by grahamehorner**
updated to use dnx.rc1-update-1 with a hack for the  A supported MIME…… type could not be found ... exception.

### CODE REVIEW

**Comment by danielgerlag** [comment]
When will this be included in a release?

**Comment by VikingsFan** [comment]
@grahamehorner can you rebase your PR in vNext branch? Thanks!

File: vNext/src/Microsoft.AspNet.OData/Extensions/ODataServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -16,15 +17,25 @@ public static class ODataServiceCollectionExtensions
         {
             services.AddScoped<ODataProperties>();
             services.AddTransient<IConfigureOptions<ODataOptions>, ODataOptionsSetup>();
-            services.ConfigureMvc(options =>
+            services.Configure<MvcOptions>(options =>
```
Comment (xuzhg) [inline vNext/src/Microsoft.AspNet.OData/Extensions/ODataServiceCollectionExtensions.cs]: Changed to: AddMvcCore

**Comment by xuzhg** [comment]
Thanks your input.

the vNext branch is migrated to RC2-final. 
 See commit here:  2b032b1. So, Have to reject this PR.

We appreciate you send us new Pull request for the contribution. Thanks.

**Pull request closed without merging**